iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -48,6 +51,7 @@ using PublisherPortUserType = iox::popo::PublisherPortUser;
 using SubscriberPortRouDiType = iox::popo::SubscriberPortRouDi;
 using SubscriberPortUserType = iox::popo::SubscriberPortUser;
 using UniquePortId = popo::TypedUniqueId<popo::BasePortData>;
+using UniqueQueueId = popo::TypedUniqueId<popo::ChunkQueueDataId_t>;
```

Comment:
@elfenpiff at first I thought this would be a good idea, but it seems that a roudi ID needs to be set before this can be used. I'm slightly leaning towards having a 64 bit unique entity ID, as suggested by you. Do you think this should be restricted to only queues or something that could also be used for other types, i.e. to never have the same IDs even if they are used for `Foo` or `Bar`?

Reply:
@elfenpiff I think we once talked about this, but I forgot the conclusion we came to

Reply:
@elBoberido a unique entity id which is independent of the type so every entity in iceoryx, port, queue, semaphore or whatever we can think of gets this unique id is the right approach in my opinion.
A uint64 provides enough room for every one so this should not be a problem and one never would like to infer the type from a given unique id so why not use one single incrementing number.